circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala

```diff
@@ -114,27 +119,31 @@ abstract class CirceAkkaSerializer[Ser <: AnyRef: ClassTag](system: ExtendedActo
     }
   }
 
-  private def scanForSerializables(packagePrefix: String): Unit = {
+  private def checkSerializableTypesForMissingCodec(packagePrefix: String): Unit = {
     val reflections = new Reflections(packagePrefix)
     val foundSerializables = reflections.getSubTypesOf(classTag[Ser].runtimeClass).asScala.filterNot(_.isInterface)
     foundSerializables.foreach { clazz =>
       try {
-        val codec = codecsMap(parents(clazz.getName))
-        assert(codec._1 != null)
-        assert(codec._2 != null)
+        codecsMap(parents(clazz.getName))
       } catch {
-        case e @ (_: NoSuchElementException | _: NotSerializableException) =>
-          log.error(
-            e,
-            s"No codec found for [{}] class. Call Register[A] for this class or its supertype and append result to codecs.",
-            clazz.getName)
-        case e: AssertionError =>
+        case e: NoSuchElementException =>
```

Comment:
Nit: maybe a regular `if` instead of try...catch?

Reply:
The condition in `if` is verbose:
```
if(!codecsMap.contains(parents.getOrElse(clazz.getName, null)))
```
I think try catch is easier to unserstand

Reply:
Ok, let's leave as is